stubs/confluent_kafka/admin/__init__.pyi

```diff
@@ -15,13 +15,14 @@ __all__ = (
     "ConfigSource",
     "NewTopic",
     "OffsetSpec",
+    "PartitionMetadata",
     "ResourceType",
     "TopicMetadata",
 )
 
 class AdminClient:
     def __init__(self, config: dict[str, str | int | Callable]) -> None: ...
-    def poll(self, timeout: float) -> int: ...
+    def poll(self, timeout: float = -1) -> int: ...
```

Comment:
```suggestion
    def poll(self, timeout: float = ...) -> int: ...
```

nit (non-blocking): Default values have no effect in type hints, and are usually omitted to eliminate duplication ([see last paragraph of this section](https://mypy.readthedocs.io/en/stable/more_types.html#function-overloading) for reference).